README.md

```diff
@@ -42,15 +42,21 @@ rpcpassword=bar
 txindex=1
 zmqpubrawblock=tcp://127.0.0.1:29000
 zmqpubrawtx=tcp://127.0.0.1:29000
+
+# lines below only needed with Bitcoin Core 0.16+
+deprecatedrpc=addwitnessaddress
+addresstype=p2sh-segwit
 ```
 
 Eclair will use any BTC it finds in the Bitcoin Core wallet to fund any channels you choose to open. Eclair will return BTC from closed channels to this wallet.
 
-On **__testnet__**, you also need to make sure that all your UTXOs are `p2sh-of-p2wpkh`.
+On **__testnet__** and wallets created with Bitcoin Core 0.15.1 and earlier, you also need to make sure that all your UTXOs are `p2sh-of-p2wpkh`.
```

Comment:
It is correct but I would phrase it differently: there are 3 options and we should explain what users will have to do in each case:

- fresh installation of Bitcoin Core 0.16: nothing to do
- migration of an existing wallets to 0.16: send all your balance to a new address
- users who stick to 0.15.1: as before